app/routes/events/view/tickets/orders/list.js

```diff
@@ -19,18 +19,49 @@ export default class extends Route.extend(EmberTableRouteMixin) {
 
   async model(params) {
     this.set('params', params);
-    const searchField = 'identifier';
     let filterOptions = [];
-    if (params.orders_status !== 'all') {
+    if (params.orders_status !== 'all' && params.search !== null) {
+      filterOptions = [
+        {
+          and: [
+            {
+              name : 'status',
+              op   : 'eq',
+              val  : params.orders_status
+            },
+            {
+              name : 'user',
+              op   : 'has',
+              val  : {
+                name : 'email',
+                op   : 'ilike',
+                val  : `%${params.search}%`
+              }
+            }
+          ]
+        }
+      ];
```

Comment:
Just push in search options if search is present. No need to create 2 more conditions like this. See the implementation of applySearchFilters